modules/level_4/cm_1005_introduction_to_programming_i/README.md

```diff
@@ -93,9 +93,9 @@ _"Specific readings for each topic are listed with direct links to free online r
 
 #### Tips & Tricks
 
-- Get familiar with finding precise coordinates in an image inside Brackets (text editor) [**note**: support for Brackets is ending on September 1, 2021. [VS Code](https://code.visualstudio.com/) is a good alternative.].
+- Each case in Sleuth has four stages. You have a maximum of **FIVE** attempts for each 'stage'. Attempt a 'stage' when you are confident. If you make a mistake, you will get a pop-up explaining what requirement you hadn't met. But if you exhaust your five attempts, fret not because, after some downtime (about 24 hours), you will be able to solve a variation of the same stage again. This will not affect your grade, and only the highest grade is counted.
+- Get familiar with finding precise coordinates in an image inside Brackets (text editor) [**note**: support for Brackets is ending on September 1, 2021. [VS Code](https://code.visualstudio.com/) is a good alternative]. If you are using VS Code, there is no easy method to view image coordinates, but a lot of students have used MS Paint for this specific task. 
```

Comment:
Both firefox and chrome have ruler in-built in the dev tools. Might be worth adding that info too ;) 


https://stackoverflow.com/questions/48490889/how-to-enable-view-of-rulers-in-chrome-devtools
https://developer.mozilla.org/en-US/docs/Tools/Rulers

Reply:
Someone actually uses MS Paint? 

Someone mentioned [Luna Paint](https://marketplace.visualstudio.com/items?itemName=Tyriar.luna-paint) extension for VS Code in Slack that supposedly can show the coordinates of the cursor.

Reply:
Yes, I saw the Luna Paint extension too. I added MS Paint because it seemed like the simplest solution to this problem. I will add the Luna Paint extension next to VS Code then. I will also add info for the built-in ruler.

Reply:
After looking at the built-in rulers, they seem like more trouble than necessary. Apparently once you enable them, you can only view the rules on the side when you hover over that specific element, and it is not dynamic. I will add the Luna Paint extension and leave it at that. Please let me know what you think @ankush.

Reply:
I guess they are kinda clunky for first-timers, luna seems like a better option. 

Anyway, sooner or later, dev tools become necessary 😁